modules/virt-cnv-network-alerts.adoc

```diff
@@ -7,6 +7,50 @@
 
```

Comment:
The assembly path in the comments should not include "html". 

Change to this:
// * virt/logging_events_monitoring/virt-virtualization-alerts.adoc